deploy/gpf_staging.tf

```diff
@@ -78,16 +80,3 @@ resource "google_cloud_run_service" "staging_backend" {
 
   depends_on = [google_secret_manager_secret_version.staging_db_connection_string, google_project_service.run]
 }
-
-resource "google_cloud_run_domain_mapping" "staging" {
-  location = var.region
-  name     = "staging.api.venturecapitol.de"
-
-  metadata {
-    namespace = var.project
-  }
-
-  spec {
-    route_name = google_cloud_run_service.staging_backend.name
-  }
-}
```

Comment:
Fehlt hier unten der Teil, der sonst in den anderen 3 Dateien (staging+production) drin ist?
Also dieser hier:
```
resource "google_cloud_run_service_iam_policy" "dlr_staging_noauth" {
  location = google_cloud_run_service.dlr_staging_backend.location
  project  = google_cloud_run_service.dlr_staging_backend.project
  service  = google_cloud_run_service.dlr_staging_backend.name

  policy_data = data.google_iam_policy.noauth.policy_data
}
```

Reply:
Stimmt, danke!